charts/exivity/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 
-version: "3.33.3"
+version: "3.33.2"
```

This will be done by our CD
```suggestion
version: "3.33.3"
```